input-explainer.md

```diff
@@ -230,11 +230,11 @@ The majority of `tracked-pointer` input sources will have a non-null `gamepad` a
 
 The WebXR Device API currently does not offer any way to retrieve renderable resources that represent the input devices from the API itself, and as such the `XRInputSource`'s `profiles` must be used to identify and load an appropriate resources. (For example, from the application's server, a CDN, or a local cache.) The `profiles` provides a list of strings that identify the device, given in descending order of preference.
 
-For example, the Samsung Odyssey controller is a variant of the standard Windows Mixed Reality controller. As a result, the `profiles` for a that controller could be:
+For example, the Samsung Odyssey controller is a variant of the standard Windows Mixed Reality controller. As a result, the `profiles` for that controller could be:
 
 ```js
 // Exact strings are examples only.
-["samsung-odyssey", "windows-mixed-reality"]
+["samsung-odyssey", "windows-mixed-reality", "touchpad-thumbstick-controller"]
```

Comment:
@thetuvix , given that this is your hardware, would you prefer the example strings to be different?

Reply:
I'm curious to hear Alex's take on this, but given the time zone differences at the moment I'm going to assume that we can safely handle any concerns that do come up as a follow-up PR.

Reply:
"Samsung Odyssey" is a headset and "Windows Mixed Reality" is a platform, so I'd probably go with `samsung-odyssey-controller` and `windows-mixed-reality-controller`, to differentiate from `windows-mixed-reality-clicker`, etc.

However, when we populate the initial registry with the identifiers we expect to be used at WebXR 1.0, we can shake out this example to match the registry then.